docker/src/main/java/clocker/docker/location/DockerContainerLocation.java

```diff
@@ -188,12 +190,13 @@ public int execScript(Map<String,?> props, String summaryForLogging, List<String
         for (String commandString : filtered) {
             parseDockerCallback(commandString);
         }
-        if (getOwner().config().get(DockerContainer.DOCKER_USE_SSH)) {
+        boolean entitySsh = Boolean.TRUE.equals(entity.config().get(DockerContainer.DOCKER_USE_SSH));
+        boolean dockerSsh = Boolean.TRUE.equals(getOwner().config().get(DockerContainer.DOCKER_USE_SSH));
+        if (entitySsh && dockerSsh) {
```

Comment:
Ah, I missed that default value when just looking at the PR changes - sounds good.